cfme/tests/infrastructure/test_timelines.py

```diff
@@ -63,15 +66,19 @@ def gen_events(delete_fx_provider_event, provider, test_vm):
     mgmt.start_vm(test_vm.name)
 
 
-def count_events(vm_name, nav_step):
+def count_events(vm, nav_step):
```

Comment:
I couldn't agree more. I'll do that but a bit later along with complete refactoring timelines tests.